bundlewrap/items/files.py

```diff
@@ -429,8 +430,31 @@ def test(self):
             ))
 
         if not self.attributes['delete'] and not self.attributes['content_type'] == 'any':
-            with self._write_local_file():
-                pass
+            with self._write_local_file() as local_path:
+                if self.attributes['test_with']:
+                    cmd = self.attributes['test_with'].format(quote(local_path))
+                    exitcode, stdout = self._run_validator(cmd)
+                    if exitcode == 0:
+                        io.debug("{i} passed local validation".format(i=self.id))
+                    elif exitcode in (126,127,255):
```

Comment:
Same as in `operations.py` lines 200-202

Reply:
```suggestion
                    elif exitcode in (126, 127, 255):
```